aws-rds-dbcluster/src/test/java/software/amazon/rds/dbcluster/CreateHandlerTest.java

```diff
@@ -116,6 +117,39 @@ public void handleRequest_CreateDbCluster_SimpleSuccess() {
         verify(rdsProxy.client(), times(3)).describeDBClusters(any(DescribeDbClustersRequest.class));
     }
 
+    @Test
+    public void handleRequest_CreateDbCluster_ServerlessV2ScalingConfiguration() {
+        when(rdsProxy.client().createDBCluster(any(CreateDbClusterRequest.class)))
+                .thenReturn(CreateDbClusterResponse.builder().build());
+        when(rdsProxy.client().addRoleToDBCluster(any(AddRoleToDbClusterRequest.class)))
+                .thenReturn(AddRoleToDbClusterResponse.builder().build());
+
+        final CallbackContext context = new CallbackContext();
+        context.setModified(true);
+
+        test_handleRequest_base(
+                context,
+                () -> DBCLUSTER_ACTIVE,
+                () -> RESOURCE_MODEL.toBuilder()
+                        .serverlessV2ScalingConfiguration(SERVERLESS_V2_SCALING_CONFIGURATION)
+                        .build(),
+                expectSuccess()
+        );
+
+        final ArgumentCaptor<CreateDbClusterRequest> captor = ArgumentCaptor.forClass(CreateDbClusterRequest.class);
+        verify(rdsProxy.client(), times(1)).createDBCluster(captor.capture());
+        verify(rdsProxy.client(), times(1)).addRoleToDBCluster(any(AddRoleToDbClusterRequest.class));
+        verify(rdsProxy.client(), times(3)).describeDBClusters(any(DescribeDbClustersRequest.class));
```

Comment:
question: why for create there are 3 describes, while for other scenarios there are only 2?